config/eslintJS.js

```diff
@@ -16,6 +16,8 @@ module.exports = {
   rules: {
     "no-console": ["error", { allow: ["warn", "error"] }],
     "prettier/prettier": "error",
+    "flowtype/require-exact-type": "error",
+    "flowtype/require-readonly-react-props": "error",
```

Comment:
It turns out that `flowtype/require-readonly-react-props` unfortunately doesn't detect props types in definition files, so it isn't useful for us, you can remove it.